manifests/params.pp

```diff
@@ -1,128 +1,131 @@
+      $package_provider   = undef
+      $collectd_dir       = '/usr/local/etc/collectd'
+      $plugin_conf_dir    = $collectd_dir
+      $service_name       = 'collectd'
+      $config_file        = '/usr/local/etc/collectd.conf'
+      $root_group         = 'wheel'
+      $java_dir           = undef
+      $default_python_dir = '/usr/local/share/collectd/python'
     }
     'OpenBSD': {
-      $package_name     = 'collectd'
-      $package_provider = undef
-      $collectd_dir     = '/etc/collectd'
-      $plugin_conf_dir  = $collectd_dir
-      $service_name     = 'collectd'
-      $config_file      = '/etc/collectd.conf'
-      $root_group       = '_collectd'
-      $java_dir         = undef
-      $python_dir       = '/usr/local/share/collectd/python'
+      $package_name       = 'collectd'
+      $package_provider   = undef
+      $collectd_dir       = '/etc/collectd'
+      $plugin_conf_dir    = $collectd_dir
+      $service_name       = 'collectd'
+      $config_file        = '/etc/collectd.conf'
+      $root_group         = '_collectd'
+      $java_dir           = undef
+      $default_python_dir = '/usr/local/share/collectd/python'
     }
     'Archlinux': {
-      $package_name     = 'collectd'
-      $package_provider = undef
-      $collectd_dir     = '/etc/collectd.d'
-      $plugin_conf_dir  = $collectd_dir
-      $service_name     = 'collectd'
-      $config_file      = '/etc/collectd.conf'
-      $root_group       = 'wheel'
-      $java_dir         = undef
-      $python_dir       = '/usr/share/collectd/python'
+      $package_name       = 'collectd'
+      $package_provider   = undef
+      $collectd_dir       = '/etc/collectd.d'
+      $plugin_conf_dir    = $collectd_dir
+      $service_name       = 'collectd'
+      $config_file        = '/etc/collectd.conf'
+      $root_group         = 'wheel'
+      $java_dir           = undef
+      $default_python_dir = '/usr/share/collectd/python'
     }
     'Gentoo': {
-      $package_name     = 'app-admin/collectd'
-      $package_provider = 'portage'
-      $collectd_dir     = '/etc/collectd.d'
-      $plugin_conf_dir  = $collectd_dir
-      $service_name     = 'collectd'
-      $config_file      = '/etc/collectd.conf'
-      $root_group       = 'collectd'
-      $java_dir         = undef
-      $python_dir       = '/usr/share/collectd/python'
+      $package_name       = 'app-admin/collectd'
+      $package_provider   = 'portage'
+      $collectd_dir       = '/etc/collectd.d'
+      $plugin_conf_dir    = $collectd_dir
+      $service_name       = 'collectd'
+      $config_file        = '/etc/collectd.conf'
+      $root_group         = 'collectd'
+      $java_dir           = undef
+      $default_python_dir = '/usr/share/collectd/python'
     }
 
     default: {
       fail("${::osfamily} is not supported.")
     }
   }
+
+  # Override with custom fact value (present only if python is installed)
+  $python_dir            = pick($::python_dir, $default_python_dir)
```

Comment:
so this changes the default python directory for at least debian from a collectd specific directory to the distrowide python directory?